src/cbc_sdk/platform/alerts.py

```diff
@@ -340,6 +340,25 @@ def _query_implementation(cls, cb, **kwargs):
         return DeviceControlAlertSearchQuery(cls, cb)
 
 
+class ContainerRuntimeAlert(BaseAlert):
```

Comment:
Did we update `VALID_ALERT_TYPES` in `BaseAlertSearchQuery`

Reply:
Oof. Getting that.

Reply:
Should be fixed now.